cypress-smoketests/integration/trial-sessions/trial-session-working-copy.spec.js

```diff
@@ -207,10 +238,6 @@ describe.skip('Judge Chambers', () => {
 
   it('views trial session working copy', () => {
     checkShowAllFilterOnWorkingCopy(testData.trialSessionIds[0]);
-    goToTrialSessionWorkingCopy({
```

Comment:
Isn't this removing what the test is supposed to verify?

Reply:
This test is being reworked, and is currently commented out until #4830 is complete. It'll be updated in the next PR.